cmd/launch.go

```diff
@@ -61,6 +63,10 @@ func init() {
 		"The auto-termination timer for the instance in minutes")
 	launchCmd.Flags().StringVarP(&flagConfig.IamInstanceProfile, "iam-instance-profile", "p", "",
 		"The profile containing an IAM role to attach to the instance")
+	launchCmd.Flags().StringVarP(&flagConfig.UserDataFilePath, "boot-script", "b", "",
+		"The absolute filepath to a bash script passed to the instance and executed after the instance starts (user data)")
+	launchCmd.Flags().StringSliceVar(&flagConfig.UserTags, "tags", nil,
+		"The tags applied to instances and volumes at launch")
```

Comment:
These tags will not be applied to other types of resources created by simple-ec2 like security group, VPC, CFN stack, is that right?

Reply:
correct-- just the instance and the volume, if any